Assets/Scenes/boxing.unity

```diff
@@ -2892,7 +2892,7 @@ MonoBehaviour:
   levelText: {fileID: 851438259}
   leftHandRenderer: {fileID: 1403299730}
   rightHandRenderer: {fileID: 328761830}
-  multiHandInteractionColor: {r: 0, g: 0, b: 0, a: 0}
+  multiHandInteractionColor: {r: 0.8352941, g: 0, b: 1, a: 1}
```

Comment:
Did you make sure that this color is the same color as the material color used for the purple punching bags?